eng/Versions.targets

```diff
@@ -84,13 +84,13 @@
     <PropertyGroup>
       <InformationalVersion>$(PackageVersion)</InformationalVersion>
       <FileVersion>$(Version).$(GitCommits)</FileVersion>
-      <AssemblyVersion>1.0.0.0</AssemblyVersion> <!-- THIS SHALL REMAIN 1.0.0.0 -->
+      <AssemblyVersion>$(GitBaseVersionMajor).$(GitBaseVersionMinor).0.0</AssemblyVersion>
     </PropertyGroup>
 
     <ItemGroup>
       <AssemblyAttribute Include="System.Reflection.AssemblyMetadata">
         <_Parameter1>Version</_Parameter1>
-        <_Parameter2>1.0.0.0</_Parameter2> <!-- THIS SHALL REMAIN 1.0.0.0 -->
+        <_Parameter2>$(GitBaseVersionMajor).$(GitBaseVersionMinor).0.0</_Parameter2>
```

Comment:
This is a scary thing and we should probably not do this.

Reply:
Can we use `<_MauiDotNetVersion>` https://github.com/dotnet/maui/blob/main/Directory.Build.props#L7  here perhaps?

Reply:
Reverted! Rather assigning a version to the build tasks as part of the merge.